pkg/storage/metrics.go

```diff
@@ -155,6 +155,9 @@ var (
 	metaSysCount = metric.Metadata{
 		Name: "syscount",
 		Help: "Count of system KV pairs"}
+	metaWritesPerSecond = metric.Metadata{
+		Name: "writes_per_second",
```

Comment:
Yeah, it's just awkward since all the other rebalancing-related metrics already fit nicely into other groups of metrics. Done, though. Look ok?